perception/radar_crossing_objects_noise_filter/README.md

```diff
@@ -0,0 +1,72 @@
+# radar_crossing_objects_noise_filter
+
+This package contains a radar noise filter module for [autoware_auto_perception_msgs/msg/DetectedObject](https://gitlab.com/autowarefoundation/autoware.auto/autoware_auto_msgs/-/blob/master/autoware_auto_perception_msgs/msg/DetectedObject.idl).
+This package can filter the noise objects which cross to the ego vehicle.
+
+## Algorithm
+
+### Background
+
+This package aim to filter the noise objects which cross from the ego vehicle.
+The reason why these objects are noise is as below.
+
+- 1. The objects which has doppler velocity can be trusted more than vertical velocity to doppler velocity.
+
+Radars can get velocity information of objects as doppler velocity, but cannot get vertical velocity to doppler velocity directory.
+Some radars can output the objects with not only doppler velocity but also vertical velocity by estimation.
+If the vertical velocity estimation is poor, it leads to output noise objects.
+In other words, the above situation is that the objects which has vertical twist viewed from ego vehicle can tend to be noise objects.
+
+The example is below figure.
+Velocity estimation fails on static objects, resulting in ghost objects crossing in front of ego vehicles.
+
+![vertical_velocity_objects](docs/vertical_velocity_objects.png)
+
+- 2. Turning around by ego vehicle affect the output from radar.
+
+When ego vehicle turn abound, radars outputting at object level sometimes fail to estimate twist of objects with compensation to ego vehicle twist.
```

Comment:
I think this mean
"When the ego vehicle turns around, the radars outputting at the object level sometimes fail to estimate the twist of objects correctly. This is due to twist compensation of the the ego vehicle." right?

Reply:
It's different from what I mean.
I fixed at [07b7501](https://github.com/autowarefoundation/autoware.universe/pull/4127/commits/07b750151231e3c287eeaf1a1d30f636ee332802), and is this clear?

Reply:
understood. LGTM.